sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md

```diff
@@ -0,0 +1,223 @@
+---
+external help file: PnP.PowerShell.dll-Help.xml
+Module Name: PnP.PowerShell
+schema: 2.0.0
+online version: https://pnp.github.io/powershell/cmdlets/Publish-PnPCompanyApp.html
+title: Publish-PnPCompanyApp
+---
+
+# Publish-PnPCompanyApp
+
+## SYNOPSIS
+
+> [!TIP]
+> We encourage you to make improvements to this documentation. Please navigate to https://github.com/pnp/powershell/blob/dev/documentation/Publish-PnPCompanyApp.md to change this file.
+
+This cmdlet publishes a new company app to Microsoft Teams.
+
+## SYNTAX
+
+```
+Publish-PnPCompanyApp -PortalUrl <String> -AppName <String> [-Description <String>] [-LongDescription <String>]
+ [-PrivacyPolicyUrl <String>] [-TermsAndUsagePolicyUrl <String>] -CompanyName <String>
+ -CompanyWebSiteUrl <String> -ColoredIconPath <String> -OutlineIconPath <String>
+ [-Force] [-NoUpload] [-Connection <PnPConnection>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet publishes a new company app (Microsoft Viva Connections) to Microsoft Teams. It will create a new package (zip file) in the current folder named after the CompanyApp value specified. E.g. if the name is 'Contoso Portal', the package will be called "Contoso Portal.zip". This package will be uploaded to the Teams App Catalog. If you do not want to upload the package automatically, e.g. prepare a package ahead of time, specify '-NoUpload'.
```

Comment:
```suggestion
This cmdlet publishes a new company app (Microsoft Viva Connections) to Microsoft Teams. It will create a new package (zip file) in the current folder named after the CompanyApp value specified. For example, if the name is 'Contoso Portal', the package will be called "Contoso Portal.zip". This package will be uploaded to the Teams App Catalog. If you do not want to upload the package automatically, for example, prepare a package ahead of time, specify '-NoUpload'.
```